accounts/abi/abi.go

```diff
@@ -91,6 +91,40 @@ func (abi ABI) Pack(name string, args ...interface{}) ([]byte, error) {
 	return append(method.ID, arguments...), nil
 }
 
+// PackOutput packs the given intf as the output of [method] to conform the ABI.
+// This does not include method ids.
```

Comment:
I agree `method` is much more clear than the `name` but let's keep it `name` since it's being used in other functions. I updated the comment accordingly.